Microsoft.Azure.CosmosRepository.Samples/ServiceTier/ExampleService.cs

```diff
@@ -1,37 +1,36 @@
-﻿// Copyright (c) IEvangelist. All rights reserved.
-// Licensed under the MIT License.
-
-using System;
-using System.Collections.Generic;
-using System.Linq.Expressions;
-using System.Threading.Tasks;
-using Microsoft.Azure.CosmosRepository;
+﻿// Copyright (c) IEvangelist. All rights reserved. Licensed under the MIT License.
 
 namespace ServiceTier
 {
+    using System;
+    using System.Collections.Generic;
+    using System.Linq.Expressions;
+    using System.Threading.Tasks;
+    using Microsoft.Azure.CosmosRepository;
```

Comment:
I prefer `using` statements to be outside the namespace, not within - please undo these changes as well.

Reply:
There was actually a reason for usings being inside the namespace historically. Here is something that explains: https://stackoverflow.com/questions/125319/should-using-directives-be-inside-or-outside-the-namespace#:~:text=Putting%20it%20inside%20the%20namespaces%20makes%20the%20declarations,whether%20they%20go%20outside%20or%20inside%20the%20namespace.

I have been putting using statements inside namespace for a really long time (10+ years) so I'll have to fight my muscle memory on fixing these if you want it that way.